Changelog.md

```diff
@@ -5,6 +5,11 @@
 This release recommends using Ruby 2.4, while retaining Ruby 2.3 as an officially supported version.
 Ruby 2.1 is no longer officially supported.
 
+## Delete public/.well-known/
+
+Before upgrading, please check if your `publlic/` folder contains a hidden `.well-known` folder.
```

Comment:
And this, ladies and gentleman, is the reason why I should not approve sentences I wrote myself!